library/src-bootstrapped/scala/internal/StringContext.scala

```diff
@@ -0,0 +1,21 @@
+package scala.internal
```

Comment:
I would put it it in the dotty package since this will be removed once we merge dotty-library and scala-library (the documentation should explain that too).

Reply:
Moved to `dotty.internal`